kiali.go

```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/kiali/kiali/server"
 	"github.com/kiali/kiali/status"
 	"github.com/kiali/kiali/util"
+
+	_ "go.uber.org/automaxprocs"
```

Comment:
just trivial issue - Kiali coding standards put third party libs in a second group (of three) of imports. So move this up in its own group on line 26, with newlines before and after.

```
...
	"strings"

	_ "go.uber.org/automaxprocs"

	"github.com/kiali/kiali/config"
...
```

I have not validated this PR at all , I just quickly noticed the coding standard violation so I'm reporting it here :)